### PR TITLE
Add opus

### PR DESCRIPTION
**Pull request by xclaesse**

### CODE REVIEW

**Comment by xclaesse** [comment]
I reported the clang-cl issue, I don't know what's going on: https://github.com/xiph/opus/issues/288.

**Comment by neheb** [comment]
Untested compiler. I added clang-cl for this reason.

**Comment by xclaesse** [comment]
@neheb is there a way to disable clang-cl from ci_config.json?

**Comment by neheb** [comment]
I did not add such a thing. My plan was to passively fix such issues. Current progress down from 43 or something:

```
10 failed: fluidsynth, glib, harfbuzz, json-glib, libffi, libuv, lzo2, mpdecimal, range-v3, spng
```

PRs for two of those are present. The first 5 are related. I don't have a solution for the others.

**Comment by tp-m** [comment]
If you can pass custom command line options for `clang-cl` you could disable asm there until it's fixed perhaps?

**Comment by eli-schwartz** [comment]
It's an upstream opus issue so I'd say merge this wrap even with failing clang-cl, as soon as an upstream ticket is opened to document the failure.

**Comment by neheb** [comment]
Yeah sure. Upstream can fix it when they please. This was the case with spng.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,11 @@ meson compile -C builddir
     Use this only when there is no other way to fix the warning.
   - `skip_dependency_check`: (Optional) List of platform specific dependency
     names that are not always provided by the project.
+  - `test_options`: (Optional) List of arguments that will be passed to
+    `meson test` command (e.g. `--timeout-multiplier`, `--no-suite`).
+  - `skip_tests`: (Optional) If set to `true` tests will not be run. This is
```
Comment (eli-schwartz) [inline README.md:67]: Are we actually using skip_tests?

Comment (xclaesse) [reply README.md:67]: I cherry-picked that commit from https://github.com/mesonbuild/wrapdb/pull/1045 where we need it.

Comment (xclaesse) [reply README.md:67]: Note we could just merge that PR first, it is also blocked by clang-cl CI not passing, and it's fixed upstream in that case, just waiting for a release.

**Comment by eli-schwartz** [comment]
> LGTM then?

Yes, sorry for forgetting to respond.

**Pull request merged**